src/FlashMessenger.php

```diff
@@ -11,50 +13,51 @@
 use Laminas\Stdlib\SplQueue;
 use ReturnTypeWillChange;
 
+use function count;
+
 /**
  * Flash Messenger - implement session-based messages
+ *
+ * @template-implements IteratorAggregate<array-key, string>
  */
 class FlashMessenger extends AbstractPlugin implements IteratorAggregate, Countable
 {
     /**
      * Default messages namespace
      */
-    const NAMESPACE_DEFAULT = 'default';
+    public const NAMESPACE_DEFAULT = 'default';
 
     /**
      * Success messages namespace
      */
-    const NAMESPACE_SUCCESS = 'success';
+    public const NAMESPACE_SUCCESS = 'success';
 
     /**
      * Warning messages namespace
      */
-    const NAMESPACE_WARNING = 'warning';
+    public const NAMESPACE_WARNING = 'warning';
 
     /**
      * Error messages namespace
      */
-    const NAMESPACE_ERROR = 'error';
+    public const NAMESPACE_ERROR = 'error';
 
     /**
      * Info messages namespace
      */
-    const NAMESPACE_INFO = 'info';
+    public const NAMESPACE_INFO = 'info';
 
-    /**
-     * @var Container
-     */
+    /** @var Container|null */
     protected $container;
 
     /**
      * Messages from previous request
-     * @var array
+     *
+     * @var array<string, SplQueue>
```

Comment:
How do I get `SplQueue<array-key, string>` without Psalm complaining about too many types? Tried `@psalm-type MessageList = SplQueue<array-key, string>` but psalm didn't like that either - I think it's because SplQueue is from stdlib as opposed to native??

Reply:
It should be possible if you reference `\SplQueue`, instead of the one provided by `laminas/laminas-stdlib` here :thinking: 

Alternatively:

`\Laminas\Stdlib\SplQueue&\SplQueue<TKey, TValue>`